File: model/host.go
package model

type HostType struct {
	Email    string `json:"email"`
	Password string `json:"password"`
	Port1    string `json:"port1"`
	Port2    string `json:"port2"`
	Image1   string `json:"image1"`
	Image2   string `json:"image2"`
}

File: public/js/host.js
$("#host-submit").on("click", (e)=>{
    e.preventDefault()
    $.post("/host", JSON.stringify({
        email:$("#email").val(),
        password:$("#password").val(),
        port1:$("#port1").val(),
        port2:$("#port2").val(),
        image1:$("#image1").val(),
        image2:$("#image2").val()
    }), (data)=>{
        console.log(data);
    })
})